### ISSUE TITLE
Nice mod , but 1.20.1 support is nonexistent :(

### ISSUE THREAD

**Issue by uczpre** [Uknownymagic/OverhauledStructures-Minecraft#1]
pls support uwu , i use create and valkyrien skies.

**Comment by Zimaya** [comment]
Absolutely! I'm currently making sure there are functional versions available for everything from 1.19 to 1.21, covering Fabric(& Quilt), Forge (& Neoforge), and Datapacks. Streamlining the updating process so I have all changes I need organised ready every time I release anything in the future too. (Although pre-1.19 would be impossible for so many reasons, else I'd take this further!)

 I completely understand the challenge of picking out mods, unable to get them to work together. :( Will not be an issue here! I'll post updates on Discord when this is done, like with all major updates. :)

**Comment by Zimaya** [comment]
1.20.1 support is up now! :)
